research/src/sources/bootstrap.json5

```diff
@@ -421,6 +421,41 @@
           "image": "bootstrap-tooltip-html.png"
         }
       ]
+    },
+    {
+      "name": "Img",
+      "openUIName": "Image",
+      "url": "https://getbootstrap.com/docs/4.0/content/images/",
+      "concepts": [
+        {
+          "name": "class-img-fluid",
+          "openUIName": "fit",
+          "description": "Makes image responsive",
+          "image": "bootstrap-image-fluid.png"
```

Comment:
This image also includes a code sample of HTML. Let's trim it down to just the component itself.